arangod/Agency/CleanOutServer.cpp

```diff
@@ -395,13 +395,22 @@ bool CleanOutServer::scheduleMoveShards(std::shared_ptr<Builder>& trx) {
            collection.hasAsChildren("shards").value().get()) {
         // Only shards, which are affected
         int found = -1;
-        int count = 0;
-        for (VPackSlice dbserver : VPackArrayIterator(shard.second->slice())) {
-          if (dbserver.copyString() == _server) {
-            found = count;
-            break;
+        if (isRepl2) {
```

Comment:
Shouldn't this be
```suggestion
        if (!isRepl2) {
```
?

Reply:
Good catch!